zebra-network/src/peer/server.rs

```diff
@@ -325,6 +325,24 @@ where
             _ => {}
         }
 
+        // Per BIP-011, since we don't advertise NODE_BLOOM, we MUST
+        // disconnect from this peer immediately.
+        match msg {
+            Message::FilterLoad { .. } => {
```

Comment:
nonessential: I am not sure whether this would actually improve legibility, but I think this could be
```
FilterLoad { .. } | FilterAdd { .. } | FilterClear { .. } => { /* ... */ }
```
since the body of each case is identical.

Reply:
Ah lovely! I was vaguely wondering if there was a nicer way to do this, thanks!